### PR TITLE
Add TDD files

### PR DESCRIPTION
**Pull request by aroquev00**
Just add the TDD project files.
I moved the portfolio into its folder and kept the TDD in another folder.

### CODE REVIEW

**Comment by seanjohnite** [comment]
ooh can you fix the build? i think you probably just have to change the clouldbuild file

**Comment by aroquev00** [comment]
Mmm like the pom.xml?

**Comment by aroquev00** [comment]
I think it fails because it does not pass the tests yet...

**Comment by aroquev00** [comment]
No wait that is not it

File: aroquev/.gitignore [hunk 1]
```diff
@@ -1 +1 @@
-target/*
+*target/*
```
Comment (brettapeters) [inline aroquev/.gitignore:1]: You can put a .gitignore in a subdirectory and it will apply to that dir & its subdirectories only.

**Pull request merged**